topsort-endpoints.yml

```
@@ -210,25 +210,25 @@ components:
           example: v_89kjm
 
     Session:
-      description: The Session object allows correlating user activity during a session whether or not they are actually logged in. For all calls to Topsort, include as many as are known.
+      description: The Session object allows correlating user activity during a session whether or not they are actually logged in.
       required:
         - sessionId
       type: object
       properties:
         sessionId:
+          description: >
+            Long-lived token identifying the customer interacting with the marketplace.
+
+            If your users are always logged in you may use a hash of your customer ID, in which case the optional `consumerId` would not be needed.
+            If your users may interact with your app or site while logged out we recommend generating a random identifier (UUIDv4) on first load and store it on local storage (cookie, local storage, etc) and let it live for at least a year.
           type: string
-          example: 345bx7
+          example: ebeaf802-6d0a-41a3-ae59-661887c4f6cb
         consumerId:
+          description: >
+            If the user is logged in, you could send us a hash of your customer ID.
```

I rephrase it as something like
```suggestion
            Optional id identifying the user, this field is required in case your marketplace wants to do cross-device attribution.
             
            Instead of sending us the exact same user id you are storing in your systems we recommend you to send us a hash of the user id (sha1, sha512, etc).
```

---

Agreed. I'll change id to ID to be consistent with the rest of the documentation though.